src/main/java/bowling/domain/Miss.java

```diff
@@ -0,0 +1,15 @@
+package bowling.domain;
+
+public class Miss extends State {
+    private int preBowl;
+
+    Miss(int firstBowl, int secondBowl) {
+        super(secondBowl, String.valueOf(secondBowl));
+        this.preBowl = firstBowl;
+    }
+
+    @Override
+    public State bowl(int firstBowl) {
+        return null;
+    }
```

Comment:
```suggestion
    @Override
    public State bowl(int firstBowl) {
        throw new IllegalBowlException();
    }
```

수행할 수 없는 메서드 콜이 일어났을 때는 null 을 리턴하지 마시고,
Custom Exception 을 던져주시면 좋겠습니다.
NullException 이 발생하는 지점이, Miss.bowl 과 멀리 떨어진 곳에서 발생하게 되면,
협업하는 동료 입장에서 디버깅하기 굉장히 어려워질 수도 있어서요.